src/quantum/index.js

```diff
@@ -530,31 +529,22 @@ export default class QuantumIndex extends React.Component {
           return null;
 
         })
-        .toArray();
+        .toArray().filter(Boolean);
```

Comment:
What is this `Boolean`?

Reply:
It helps filter out null items:
http://www.devign.me/javascript-tip-remove-falsy-items-out-of-an-array